src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/CancelFlow.tsx

```diff
@@ -64,11 +64,26 @@ export const CancelFlow = (props: Props) => {
                   : "settings-unsubscribe-dialog-confirmation-redirect-title",
             )}
             illustration={
-              <Image
-                className={styles.cancellationIllustrationWrapper}
-                src={CancellationSurveyPlaneIllustration}
-                alt=""
-              />
+              <video
+                aria-hidden={true}
+                autoPlay={true}
+                loop={true}
+                muted={true}
+                className={styles.cancellationAnimation}
+              >
+                <source
+                  // Unfortunately video files cannot currently be imported, so make
+                  // sure these files are present in /public. See
+                  // https://github.com/vercel/next.js/issues/35248
+                  src="/animations/CancellationFlowAnimation.mp4"
+                  type="video/mp4"
+                />
+                <Image
+                  className={styles.cancellationIllustrationWrapper}
+                  src={CancellationSurveyPlaneIllustration}
+                  alt=""
+                />
```

Comment:
I'm not seeing the fallback image if I disable animations... You'll probably need something like this: https://github.com/mozilla/fx-private-relay/blob/a515dc1c69e76a40c773c92d27af3ba485ea1e5e/frontend/src/components/layout/navigation/whatsnew/WhatsNewContent.module.scss#L32-L42